gfx/common/x11_common.c

```diff
@@ -306,9 +306,17 @@ static void x11_suspend_screensaver_xdg_screensaver(Window wnd, bool enable)
 
 void x11_suspend_screensaver(Window wnd, bool enable)
 {
-    x11_suspend_screensaver_xdg_screensaver(wnd, enable);
 #ifdef HAVE_DBUS
     x11_suspend_screensaver_dbus(enable);
+    if (dbus_screensaver_cookie == 0)
+    {
+        // Only attempt xdg-screensaver method if dbus didn't succeed
+#endif
+
+        x11_suspend_screensaver_xdg_screensaver(wnd, enable);
+
+#ifdef HAVE_DBUS
+    }
```

Comment:
Let's remove the braces on that if statement. That way, the other ifdef disappears.

Pretty sure this PR doesn't properly handle reenabling the screensaver, either.